index.bs

```diff
@@ -2215,17 +2218,17 @@ The <dfn method for=IDBFactory>open(|name|, |version|)</dfn> method steps are:
 
 1. Let |environment| be [=/this=]'s [=/relevant settings object=].
 
-1. Let |origin| be |environment|'s [=environment settings object/origin=].
+1. Let |storageKey| be the result of running [=obtain a storage key=] given |environment|.
 
-1. If |origin| is an [=opaque origin=], [=throw=] a
+1. If |storageKey|'s origin is an opaque, [=throw=] a
```

Comment:
Change "an opaque" to "opaque" ? Also, can we [link the term](https://html.spec.whatwg.org/multipage/origin.html#concept-origin-opaque) while we're here?